packages/store-sync/src/postgres-decoded/index.ts

```diff
@@ -1,5 +1,4 @@
 export * from "./buildTable";
-export * from "./cleanDatabase";
```

Comment:
this is done at the `store-indexer` level when the process starts up (rather than at the store sync level) and there's currently no binary/command for `postgres-decoded`